app/controllers/challengeCtrl.js

```diff
@@ -1,5 +1,14 @@
 'use strict'
 
-app.controller('challengeCtrl', function($scope) {
-  
+app.controller('challengeCtrl', function($scope, challengeFactory) {
+
+  challengeFactory.loadChallenge($scope.gameState.challengeNum)
+  .then((challengeObj) => {
+    // need to work out how to load in correct answer images
+    $scope.originImage = challengeObj.originImage.initial
```

Comment:
Lets put everything on an object on the $scope to keep it namespaced.

So, just do like:

    $scope.challengeData = challengeObject;

To address that comment up there. We will go ahead an load the origin/local correct/incorrect images onto the scope, then we'll use an a ng-if inside the template to decide whether to show or hide them.